### PR TITLE
[New Sample] Bot Streaming in C# for Microsoft Teams

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            };
+
+            if (!string.IsNullOrEmpty(text))
+            {
+                streamingActivity.Text = text;
+            }
+
+            /*
+             * We are sending the final streamed mesage as an Adaptive Card Attachment built 
+             * using a templated.
```
Comment (SolangeAO) [inline samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs:191]: nit: "template".

File: samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                };
+                var attachment = new Attachment()
+                {
+                    ContentType = "application/vnd.microsoft.card.adaptive",
+                    Content = JsonConvert.DeserializeObject(template.Expand(tempData)),
+                };
+                
+                streamingActivity.Attachments = new List<Attachment>() { attachment };
+
+                //Add text to the activuty
```
Comment (SolangeAO) [inline samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs:209]: nit: "activity"

File: samples/bot-streaming/csharp/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    - Also, set MicrosoftAppType in the `appsettings.json`. (**Allowed values are: MultiTenant(default), SingleTenant, UserAssignedMSI**)
+
+1) Run your bot, either from Visual Studio with `F5` or using `dotnet run` in the appropriate folder.
+
+1) __*This step is specific to Teams.*__
+    - **Zip** up the contents of the `appPackage` folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    - **Upload** the `manifest.zip` to Teams (In Teams Apps/Manage your apps click "Upload an app". Browse to and Open the .zip file. At the next dialog, click the Add button.)
+    - Add the app to personal scope (Supported scopes)
+    - With CTRL + Alt + Shift + 8 open the UI dev tools pane. Go to "Settings" and search `enablebotstreaming`. Enable the flag and reload the page for the UI to reflect the changes and allow UI to reflect streaming feature changes.
```
Comment (SolangeAO) [inline samples/bot-streaming/csharp/README.md:107]: This line can be now removed.

File: samples/bot-streaming/csharp/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+**Note**: If you are facing any issue in your app, please uncomment [this](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/bot-streaming/csharp/AdapterWithErrorHandler.cs#L25) line and put your debugger for local debug.
+
+## Running the sample
+
+You can interact with this bot in Teams by sending it a message, or selecting a command from the command list. The bot will respond to the following strings.
```
Comment (SolangeAO) [inline samples/bot-streaming/csharp/README.md:113]: Since this bot does not have any commands, maybe we can remove this line

**Review by SolangeAO — APPROVED** [review]

File: samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-streaming/csharp/Bots/TeamsConversationBot.cs:191]: I have updated " using a template."

File: samples/bot-streaming/csharp/README.md (hunk 4 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-streaming/csharp/README.md:113]: I have removed link 113.

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11839526197

image

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]
Looks good ! approving.

**Pull request merged**